frontend/src/locales/es.json

```diff
@@ -198,6 +198,7 @@
     "decay-14-day": "Disminución gradual por 14 días",
     "delete-the-link": "Eliminar el enlace?",
     "deleted": "El enlace ha sido eliminado!",
+    "disbured": "Disbursement of the Link-Ammount initiated! The transfer of {n} GDD into your account will be completed shortly.",
```

Comment:
If you don't want to translate it, please don't add it, it will automatically fall back to English.